packages/bundler-plugin-core/src/types.ts

```diff
@@ -93,13 +97,17 @@ export type Options = Omit<IncludeEntry, "paths"> & {
   /**
    * Version control system remote name.
    *
+   * This value can also be set via the `SENTRY_VSC_REMOTE` env variable.
+   *
    * Defaults to 'origin'.
    */
   vcsRemote?: string;
 
   /**
    * A header added to every outgoing network request.
    * The format should be `header-key: header-value`.
+   *
+   * This value can also be set via the `CUSTOM_HEADER` env variable.
```

Comment:
M: Is there a reason why this isn't `SENTRY_CUSTOM_HEADER`? Also, can you set more than one header here?

Reply:
I was also weirded out by this inconsistent naming but the env variable is called `CUSTOM_HEADER` in Sentry CLI: https://docs.sentry.io/product/cli/configuration/#configuration-values

So we can't really change that as long as we want to maintain compatibility and I'd vote that just for this purpose introducing a new variable in the plugin which differs from the CLI causes more confusion than it brings value

Reply:
Judging by the CLI documentation,

> The header that will be added to every outgoing request in key:value format

I don't think you could add more than one. Although I'm not sure what the CLI would do with a value like `key1:value1\nkey2:value2`.

Reply:
To avoid future deprecation possibility, my gut would be to a) make both work for now, and b) only document that one with `SENTRY` in the name. That way, if/when you do ever move away from using `sentry-cli`, you're not stuck with its legacy weird choices.

UPDATE: We weren't the only ones to have this idea: https://github.com/getsentry/sentry-cli/blob/a08246db0f52438b0c32886148502f6be2591db4/CHANGELOG.md?plain=1#L221

So if we go to `sentry-cli` 2.0, this will be a moot point.

Reply:
I extracted this into an issue (#106). Would you mind giving it a read @lobsterkatie?